src/RayMarchedAtlasVolume.ts

```diff
@@ -58,9 +59,8 @@ export default class RayMarchedAtlasVolume implements VolumeRenderImpl {
     this.scale = new Vector3(1.0, 1.0, 1.0);
     this.isOrtho = false;
 
-    this.cube = new BoxGeometry(1.0, 1.0, 1.0);
-    this.cubeMesh = new Mesh(this.cube);
-    this.cubeMesh.name = "Volume";
+    this.uniforms = rayMarchingShaderUniforms;
```

Comment:
Do we copy `rayMarchingShaderUniforms` anywhere or will all instances of this class that may exist share the same reference? (Potentially niche issue and also one that predates this PR if it exists at all)

Reply:
Yeah I think this has been a potential issue all along - might want to check if ShaderMaterial makes its own copy of the uniforms objects passed in, or else we should clone it when passing it in.

Reply:
Because we only ever have one of these active at any time, it hasn't ever been a concern.

Reply:
Fixed based on our discussion, thank you!